vm/src/stdlib/os.rs

```diff
@@ -262,27 +262,7 @@ impl IntoPyException for &'_ io::Error {
 #[cfg(unix)]
 impl IntoPyException for nix::Error {
     fn into_pyexception(self, vm: &VirtualMachine) -> PyBaseExceptionRef {
-        match self {
-            nix::Error::InvalidPath => {
-                let exc_type = vm.ctx.exceptions.file_not_found_error.clone();
-                vm.new_exception_msg(exc_type, self.to_string())
-            }
-            nix::Error::InvalidUtf8 => {
-                let exc_type = vm.ctx.exceptions.unicode_error.clone();
-                vm.new_exception_msg(exc_type, self.to_string())
-            }
```

Comment:
is this branch covered by `io::Error::from`?

Reply:
In 0.22, if something needs to validate utf8 it'll return something other than a nix::Result.